docker-images/al2/Dockerfile

```diff
@@ -0,0 +1,89 @@
+ARG AL_VERSION=2
+
+FROM amazonlinux:${AL_VERSION}
+
+WORKDIR /opt/s2n
+
+ARG OPENSSL_VERSION=openssl-1.1.1
+ARG GCC_VERSION=7
+ARG ZSH_THEME=cypher
+
+ENV S2N_LIBCRYPTO=${OPENSSL_VERSION} \
+    GCC_VERSION=${GCC_VERSION} \
+    BUILD_S2N=true \
+    TESTS=integration \
+    EPEL_REPO="https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm"
+
+# The `s2n_setup_env` assumes bash, not sh
+SHELL ["/bin/bash", "-c"]
+
+# set up user account
+RUN set -ex \
+    && amazon-linux-extras install ruby2.6 rust1 \
```

Comment:
running `set -ex` at the top removes the need to chain `&&` everywhere. you can just put a `;` at the end of the line instead.